<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>580</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>730</width>
    <height>580</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>730</width>
    <height>580</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>730</width>
    <height>580</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Shader Library</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_Pipe">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>731</width>
      <height>71</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <widget class="QComboBox" name="cbox_categorys">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>180</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>180</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="btn_browseCategory">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>10</y>
       <width>48</width>
       <height>48</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="statusTip">
      <string>Browse Category Folder</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="iconSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="btn_addCategory">
     <property name="geometry">
      <rect>
       <x>551</x>
       <y>10</y>
       <width>48</width>
       <height>48</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="statusTip">
      <string>Create New Category Folder</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="btn_favoriteCat">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>48</width>
       <height>48</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="statusTip">
      <string>Pin Category</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="iconSize">
      <size>
       <width>28</width>
       <height>32</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="btn_refresh">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>10</y>
       <width>48</width>
       <height>48</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="statusTip">
      <string>Refresh Category Shaders</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="iconSize">
      <size>
       <width>28</width>
       <height>32</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="btn_unPinTab">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>10</y>
       <width>48</width>
       <height>48</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="statusTip">
      <string>UnPin Category</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="iconSize">
      <size>
       <width>28</width>
       <height>32</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tab_materials">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>501</width>
      <height>461</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QToolButton" name="btn_saveChanges">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>480</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Save Changes</string>
    </property>
    <property name="statusTip">
     <string>Save Shader Notes</string>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>SAVE</string>
    </property>
    <property name="iconSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="popupMode">
     <enum>QToolButton::InstantPopup</enum>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonIconOnly</enum>
    </property>
    <property name="autoRaise">
     <bool>false</bool>
    </property>
    <property name="arrowType">
     <enum>Qt::NoArrow</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>80</y>
      <width>211</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Shader Info</string>
    </property>
    <widget class="QLabel" name="lbl_shaderName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ShaderName</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_shaderUserPC">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>UserInfo</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_shaderType">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ShaderType</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_shaderCode">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>CODE</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="te_notes">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>300</y>
      <width>211</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="btn_addShader">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>480</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
    <property name="statusTip">
     <string>Add Selected Shader</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="autoRaise">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>730</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="mnu_reloadCategorys"/>
    <addaction name="separator"/>
    <addaction name="mnu_reset"/>
   </widget>
   <widget class="QMenu" name="menuCategory">
    <property name="title">
     <string>Category</string>
    </property>
    <addaction name="mnu_addNewCategory"/>
    <addaction name="separator"/>
    <addaction name="mnu_browseCategoryFolder"/>
   </widget>
   <widget class="QMenu" name="menuShader">
    <property name="title">
     <string>Shader</string>
    </property>
    <addaction name="actionExport_Selected_Shader"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="separator"/>
    <addaction name="mnu_defaultLigthRigOpen"/>
    <addaction name="mnu_setFolder"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuCategory"/>
   <addaction name="menuShader"/>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="mnu_addNewCategory">
   <property name="text">
    <string>Add New Category</string>
   </property>
  </action>
  <action name="mnu_browseCategoryFolder">
   <property name="text">
    <string>Browse Category Folder</string>
   </property>
  </action>
  <action name="actionExport_Selected_Shader">
   <property name="text">
    <string>Collect Maps</string>
   </property>
  </action>
  <action name="mnu_reset">
   <property name="text">
    <string>Reset Tool</string>
   </property>
  </action>
  <action name="actionOverwrite_Confirmation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Overwrite Confirmation</string>
   </property>
  </action>
  <action name="mnu_defaultLigthRigOpen">
   <property name="text">
    <string>Open Thumbnail LightRig File</string>
   </property>
  </action>
  <action name="mnu_reloadCategorys">
   <property name="text">
    <string>Reload Categorys</string>
   </property>
  </action>
  <action name="mnu_setFolder">
   <property name="text">
    <string>Set Shaders Storage Folder</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
